### PR TITLE
prepare release 0.9.0

### PR DESCRIPTION
**Pull request by cehbrecht**
### Pull Request Checklist:
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes issue #xyz
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)
- [x] HISTORY.rst has been updated (with summary of main changes)
- [x] I have added my relevant user information to `AUTHORS.md`

* **What kind of change does this PR introduce?:** 

Preparing release 0.9.0

* **Does this PR introduce a breaking change?:** 

no

* **Other information:**

### CODE REVIEW

**Comment by Zeitsperre** [comment]
@cehbrecht Will this include #216 or is this a tag for work-to-date before integrating it?

**Comment by cehbrecht** [comment]
> Will this include #216 or is this a tag for work-to-date before integrating it?

@Zeitsperre Ag has merged #216. I want to make a release of the full rook stack this week.

**Comment by cehbrecht** [comment]
@sol1105 I want to release clisops late tomorrow (Wednesday). You can still merge PRs to master. It should just not break subset and average operators. We can still make patch releases after easter.

**Comment by cehbrecht** [comment]
@sol1105 @Zeitsperre @agstephens should we merge #217?

**Comment by cehbrecht** [comment]
@Zeitsperre @agstephens @sol1105 Good to go? I would merge later then and make the release.

File: HISTORY.rst [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 ===============
 
-v0.9.0 (unreleased)
+v0.9.0 (2022-04-13)
 -------------------
 
 New Features
 ^^^^^^^^^^^^
-* ``clisops`` now uses the `loguru <https://loguru.readthedocs.io/en/stable/index.html>`_ library as its primary logging engine.
+
```
Comment (Zeitsperre) [inline HISTORY.rst:9]:
```suggestion
```

**Review by Zeitsperre — APPROVED** [review]

**Pull request merged**